seqr/views/apis/users_api_tests.py

```diff
@@ -9,29 +9,14 @@
 from seqr.views.apis.users_api import get_all_collaborators, set_password, create_staff_user, \
     create_project_collaborator, update_project_collaborator, delete_project_collaborator, forgot_password, \
     get_all_staff
-from seqr.views.utils.test_utils import AuthenticationTestCase
+from seqr.views.utils.test_utils import AuthenticationTestCase, AnvilAuthenticationTestCase,\
+    MixAuthenticationTestCase, WORKSPACE_FIELDS
 
 
 PROJECT_GUID = 'R0001_1kg'
 
 
-class UsersAPITest(AuthenticationTestCase):
```

Comment:
Why split this test into a separate test case? I would expect this test to behave the same for all different modes of authentication, so why not leave it in the base `UsersAPITest` class and let it run with all 3 authentications to confirm it always works correctly?

Reply:
I thought the user creating/updating should be disabled when AnVIL is enabled. But, you are right, we need these features for the `mix` cases at least.